vault.js

```diff
@@ -1,4 +1,21 @@
 'use strict';
 module.exports = function() {
-  
+  var obj = {};
```

Comment:
try to us descriptive variable name. The name `vault` or `store` would give a bit more insight into its use. Obj is pretty non-descript. It is ok to add Obj to a variable name to help identify its data type. eg. `vaultObj`